src/main/java/uk/gov/hmcts/reform/hmc/model/HearingWindow.java

```diff
@@ -3,16 +3,11 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-import java.time.LocalDate;
-import java.time.LocalDateTime;
-
 @Data
 @NoArgsConstructor
 public class HearingWindow {
 
-    private LocalDate hearingWindowStartDateRange;
-
-    private LocalDate hearingWindowEndDateRange;
+    private HearingWindowDateRange hearingWindowDateRange;
```

Comment:
were these refactors part of the ticket and scope?

Reply:
No. Should have been a separate ticket.